src/test/java/uk/gov/dhsc/htbhf/steps/ManualAddressSteps.java

```diff
@@ -0,0 +1,160 @@
+package uk.gov.dhsc.htbhf.steps;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import uk.gov.dhsc.htbhf.page.ManualAddressPage;
+import uk.gov.dhsc.htbhf.page.PageName;
+
+import static uk.gov.dhsc.htbhf.steps.ClaimValuesTestDataFactory.buildClaimValuesForManualAddress;
+import static uk.gov.dhsc.htbhf.steps.Constants.*;
+
+/**
+ * Steps for manually enter an address
+ */
+public class ManualAddressSteps extends CommonSteps {
+
+    private static final String BLANK_STRING = "";
+
+    @Given("^I have entered my details and wish to manually enter my address")
+    public void enterDetailsUpToPage() {
```

Comment:
Couldn't use the generic step as we need to make sure that `selectAddress` is false in the `ClaimValues` so we can navigate to the manual address page.